boards/nrf52840dongle/reset.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C)  2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dongle
+ * @{
+ * @file
+ * @brief       Implementation for managing the nrfutil bootloader
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#ifdef MODULE_USB_BOARD_RESET
+
+#define USB_H_USER_IS_RIOT_INTERNAL
+
+#include "usb_board_reset.h"
+
+/* Set the value used by the bootloader to select between boot in
+   application and boot in bootloader mode. */
+#define NRF52_DOUBLE_TAP_MAGIC_NUMBER       (0x4e)
```

Comment:
Now we only have to find out where that number comes from.
This is from `adafruit-clue`, while `arduino-nano-33-ble` uses `0xb0`.

What's the pattern? We should move this to a common location when we find out :wink: